Exploration Queries/SecurityEvent/ParentProcessesOnHost.txt

```diff
@@ -23,6 +22,7 @@ and ParentProcessName !contains ":\\Windows\\CCM\\CcmExec.exe"
 // excluding well known parent process, child process combinations, feel free to add more specific to the environment
 | where (ParentProcessName !contains ":\\Windows\\System32\\svchost.exe" and (NewProcessName !contains ":\\Windows\\System32\\wbem\\WmiPrvSE.exe" or NewProcessName !contains ":\\Windows\\SysWOW64\\wbem\\WmiPrvSE.exe"))
 | where (ParentProcessName !contains ":\\Windows\\System32\\services.exe" and NewProcessName !contains ":\\Windows\\servicing\\TrustedInstaller.exe")
+| where toupper(Computer) contains v_Host_HostName or toupper(WorkstationName) contains v_Host_HostName
```

Comment:
lines of the format (toupper(), tolower(), etc...) are computationally heavier than other simple clauses. So it is good to try and push them as much further down the query as possible so they do not affect the performance of the query